lms/djangoapps/badges/models.py

```diff
@@ -132,6 +132,7 @@ class BadgeAssertion(models.Model):
     backend = models.CharField(max_length=50)
     image_url = models.URLField()
     assertion_url = models.URLField()
+    created_at = models.DateTimeField(auto_now_add=True, db_index=True)
```

Comment:
Consider using [TimeStampedModel](http://django-model-utils.readthedocs.org/en/latest/models.html#timestampedmodel) instead from `model_utils.models`.  You can see other uses of it in edx-platform.